lib/interfaces.js

```diff
@@ -111,11 +119,19 @@ function httpsFactory(iface, fallback, mosca) {
   }
 
   if (credentials.keyPath) {
-    credentials.key = fs.readFileSync(credentials.keyPath);
+    if (fs.existsSync(credentials.keyPath)) {
+      credentials.key = fs.readFileSync(credentials.keyPath);
+    } else {
+      credentials.key = credentials.keyPath;
+    }
   }
 
   if (credentials.certPath) {
-    credentials.cert = fs.readFileSync(credentials.certPath);
+    if (fs.existsSync(credentials.certPath)) {
+      credentials.cert = fs.readFileSync(credentials.certPath);
+    } else {
+      credentials.cert = credentials.certPath;
+    }
```

Comment:
Yes, it should be that way. We should prefer `key` and `cert` to `keyPath` and `certPath`. They are legacy API, unfortunately.